### PR TITLE
Update theme constants for new ThemeBuilder

### PR DESCRIPTION
**Pull request by groshenkovamarina**
Removed some constants, because of they were outdated or not needed, you can see them in the file in this pull request (migration-metadata.js), in case of tree list, it uses data grid constants, also added new constants, for tabs, navbars and not only. Changed the sorting in the groups, changed the group in some constants.

### CODE REVIEW

**Comment by nightskylark** [comment]
No one from five reviewers has left a review for a week. It looks like nobody knows how to review it. Maybe some additional description could help?

**Comment by BingoRUS** [comment]
@nightskylark some internal feedback was left, we need to prepare a migration mechanism to avoid the possible BC. But I agree that a detailed description would be helpful.

File: themebuilder/tests/migration-metadata-loader-spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "Key": "@treelist-border-color"
+}];
+
+describe("Migration metadataLoader", () => {
+    beforeEach(() => {
+        mock("../data/metadata/dx-theme-builder-metadata", metadata);
+        mock("../data/migration-metadata/migration-metadata", migrationData);
+    });
+
+    it("Loadmetadata", () => {
```
Comment (babich-a) [inline themebuilder/tests/migration-metadata-loader-spec.js]:
```suggestion
    it("Load metadata", () => {
```

File: themebuilder/modules/metadata-loader.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
         this.fullMetadata = require("../data/metadata/dx-theme-builder-metadata");
+        this.migrationMetadata = require("../data/migration-metadata/migration-metadata");
     }
 
     load(theme, colorScheme) {
         return new Promise((resolve) => {
             let metadata = this.fullMetadata[theme + "_" + colorScheme.replace("-", "_") + "_metadata"];
+
+            if(semver.gte(this.version(), "18.2.8") && this.migrationMetadata[theme]) {
+                for(let i = 0; i < this.migrationMetadata[theme].length; i++) {
```
Comment (babich-a) [inline themebuilder/modules/metadata-loader.js]:
```suggestion
                this.migrationMetadata[theme].forEach(value => {
                    metadata.push({ "Key": value });
                });
```

**Review by babich-a — CHANGES_REQUESTED** [review]

File: styles/widgets/material/accordion.material.less [hunk 3]
```diff
@@ -32,11 +32,17 @@
         border-top: none;
     }
 
-    &.dx-state-active, &.dx-state-focused {
+    &.dx-state-active, &.dx-state-focused {
```
Comment (San4es) [inline styles/widgets/material/accordion.material.less]:
```suggestion
    &.dx-state-active, &.dx-state-focused {
```

File: styles/widgets/material/calendar.material.less [hunk 4]
```diff
@@ -278,7 +278,7 @@
     &.dx-calendar-selected-date {
         &, &.dx-calendar-today {
             span {
-                color: @base-selected-text-color;
+                color: @base-inverted-text-color;;
```
Comment (San4es) [inline styles/widgets/material/calendar.material.less]:
```suggestion
                color: @base-inverted-text-color;
```

**Review by babich-a — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

File: styles/widgets/generic/color-schemes/light/generic.light.less [hunk 5]
```diff
@@ -1224,401 +1005,206 @@
...
+@datagrid-columnchooser-shadow-color: fade(@base-shadow-color, 20%);
+
+@datagrid-columnchooser-overlay-border-radius: @base-border-radius-large; // TODO: may be not need
+
+@datagrid-column-separator-bg: fade(@base-accent, 50%);
+@datagrid-text-stub-background-image-path: url(images/widgets/generic/color-schemes/light/grid/text-stub.png);
+
+// TreeList
+
+@treelist-base-color: @base-text-color;
```
Comment (vconst) [inline styles/widgets/generic/color-schemes/light/generic.light.less:1162]: Why is not possible to customize Tree List styles?

Comment (vconst) [reply styles/widgets/generic/color-schemes/light/generic.light.less:1162]: When Tree List was implemented we decided that it should have separate settings in theme builder.

Comment (groshenkovamarina) [reply styles/widgets/generic/color-schemes/light/generic.light.less:1162]: It is the designer's decision because they had the same settings.

**Review by BingoRUS — APPROVED** [review]

**Pull request merged**